tests/align/test_align_global_alignment_specialized.h

```diff
@@ -315,4 +315,33 @@ SEQAN_DEFINE_TEST(test_align_global_alignment_score_myers_hirschberg)
     SEQAN_ASSERT_EQ(res, -8);
 }
 
+SEQAN_DEFINE_TEST(test_align_global_alignment_gotoh_score_types)
```

Comment:
Could you a) please add a general test case for the correct infinity value for floats and doubles?
See here: https://github.com/seqan/seqan/blob/5e8ca158982943fbc3398a192ece9e849c904d1f/tests/align/test_alignment_dp_cell.h#L75-L89

And could you move this test to https://github.com/seqan/seqan/blob/develop/tests/align/test_align_global_alignment.h
Basically at the end and call the test: `test_align_global_alignment_fix_floating_point_scores`

Reply:
sure!